ReactiveCocoa/NSObject+Association.swift

```diff
@@ -0,0 +1,22 @@
+import ReactiveSwift
+
+extension Reactive where Base: NSObject {
+	/// Retrieve the associated value for the specified key. If the value does not
+	/// exist, `initial` would be called and the returned value would be
+	/// associated subsequently.
+	///
+	/// - parameters:
+	///   - key: An optional key to differentiate different values.
+	///   - initial: The action that supples an initial value.
+	///
+	/// - returns:
+	///   The associated value for the specified key.
+	internal func associatedValue<T>(forKey key: StaticString = #function, initial: (Base) -> T) -> T {
+		var value = objc_getAssociatedObject(base, key.utf8Start) as? T
```

Comment:
Is it necessary to extract it, when `key` is an immutable owner of the pointer and `utf8Start` does not have a non-trivial cost behind its getter?